common/constant/key.go

```diff
@@ -416,3 +416,10 @@ const (
 	MetricsMetadata = "dubbo.metrics.metadata"
 	MetricApp       = "dubbo.metrics.app"
 )
+
+// default meta cache config
+const (
+	DefaultMetaCacheName = "meta"
+	DefaultMetaFileName  = ".matadata"
```

Comment:
This name is too common to use. Considering adding a prefix, `dubbogo.metadata` to avoid name conflict. Btw, there is a typo:

```suggestion
	DefaultMetaFileName  = ".metadata"
```